functions.php

```diff
@@ -36,4 +36,22 @@ function register_my_menu() {
   register_nav_menu('header-menu',__( 'Header Menu' ));
 }
 add_action( 'init', 'register_my_menu' );
+?>
+
+<?php
+add_filter( 'wp_title', 'wpdocs_hack_wp_title_for_home' );
```

Comment:
Looking good, just make sure your function names are meaningful. `wpdocs_hack_wp_title_for_home` isn't really representative for what you're doing.